local/languages/finna/fi.ini

```diff
@@ -1864,3 +1873,8 @@ kavi_notification = "Ilmoitus"
 Family Name = "Suku"
 Personal Name = "Henkilö"
 Corporate Name = "Yhteisö"
+
+
+; Record label tooltips
+RecordLabelTooltip:r2_restricted_metadata_available_html = "<h4>Sisältää pääsyrajoitettuja henkilötietoja</h4><p>Aineistotiedot sisältävät pääsyrajoitettua tietoa. <a href="%%register%%" data-lightbox>Kirjaudu Tutkijasaliin</a> saadaksesi pääsyn. <a href="%%terms%%" data-lightbox>Aineistotiedon käyttöehdot</a></p>"
```

Comment:
Kaksoispiste viittaa erilliseen namespaceen. Jos näin, niin omaan tiedostoonsa. Jos ei, niin kaksoispisteestä eroon.